subprojects/core/src/main/java/org/gradle/initialization/StartParameterBuildOptions.java

```diff
@@ -288,6 +289,23 @@ public void applyTo(boolean value, StartParameterInternal settings, Origin origi
         }
     }
 
+    public static class WatchFileSystemOption extends BooleanBuildOption<StartParameterInternal> {
+        public static final String LONG_OPTION = "watch-fs";
+        public static final String GRADLE_PROPERTY = "org.gradle.unsafe.watch-fs";
+
+        public WatchFileSystemOption() {
+            super(GRADLE_PROPERTY, BooleanCommandLineOptionConfiguration.create(
+                LONG_OPTION,
+                "Enables watching the file system for changes, allowing data about the file system to be re-used for the next build.",
+                "Disables re-using data about the file system for the next build."
```

Comment:
Should we mention that this prevents the file system from being watched? After all, this is going to be enabled by default later on.

Reply:
I changed the message to
```
Disables watching the file system.
```